glusterd2/xlator/options/options.go

```diff
@@ -236,7 +242,17 @@ func ValidatePriorityOrSize(o *Option, val string) error {
 
 // ValidateSizeList validates if the option is a valid size list
 func ValidateSizeList(o *Option, val string) error {
-	return ErrInvalidArg
+	l := len(val)
+	if strings.ContainsRune(val, 'B') {
+		t := strings.TrimSpace(val[:l-2])
+		v, err := strconv.ParseInt(t, 10, 64)
+		if err != nil {
+			return err
+		} else if v%512 != 0 {
+			return ErrInvalidArg
+		}
+	}
+	return nil
 }
 
 // ValidateSizet validates if the option is a valid size
```

Comment:
I noticed this from the corner of my eye.

The check below on line [264][1], should be `if o.Min == 0 && o.Max == 0`. But you should be using the common `ValidateRange` function instead.

[1]: https://github.com/gluster/glusterd2/pull/500/files#diff-acbeafc2d6d1587badec6a4f81ee69daR264